app/experimenter/templates/experiments/edit_variants.html

```diff
@@ -19,18 +19,17 @@
       <h4 class="col-9 offset-3">Firefox Population</h4>
     </div>
 
-
     {% include "experiments/field_inline.html" with field=form.population_percent %}
 
-    {% include "experiments/field_inline.html" with field=form.firefox_channel %}
+    {% include "experiments/field_inline_help_link.html" with field=form.firefox_channel %}
 
     <div class="form-group row">
       <div class="col-3 text-right">
         <label class="control-label">
           <strong>Firefox Versions</strong>
         </label>
-        <br/>
-        <a href="#population-percent-help" data-toggle="collapse" aria-expanded="false" aria-controls="population-percent-help">Help</a>
```

Comment:
Oh do we lose the population percent help text?

Reply:
@jaredkerim no we don't lose it... check out what's in `<strong>` .... this is actually in the firefox versions section. I think I accidentally left this in when copying and pasting when making this new FF versions section.... that's why I HAD to do this ticket... I noticed this random line of code here earlier in the week when working on something else.